tests/installation/welcome.pm

```diff
@@ -95,14 +95,15 @@ sub run {
         # In upgrade mode, there is no product list shown in welcome screen
         unless (check_var('ARCH', 's390x') || get_var('UPGRADE')) {
             assert_screen('select-product');
-            my %hotkey = (
-                sles     => 's',
-                sled     => 'u',
-                sles4sap => get_var('OFW') ? 'u' : 'i',
-                hpc      => check_var('ARCH', 'x86_64') ? 'x' : 'u'
-            );
+            # my %hotkey = (
+            #     sles     => 's',
+            #     sled     => 'u',
+            #     sles4sap => get_var('OFW') ? 'u' : 'i',
+            #     hpc      => check_var('ARCH', 'x86_64') ? 'x' : 'u'
+            # );
             my $product = get_required_var('SLE_PRODUCT');
-            send_key 'alt-' . $hotkey{$product};
+            # send_key 'alt-' . $hotkey{$product};
+            assert_and_click('before-select-product-' . $product);
```

Comment:
Be aware that we also have textmode installations and hence need to use shortcuts there.

Reply:
We can also handle it with needles.